packages/core/src/lib/store/index.ts

```diff
@@ -408,7 +404,7 @@ export async function createCursor(
     digest,
     pubsubTopic,
     senderTime: messageTime,
-    receivedTime: messageTime,
+    receiverTime: messageTime,
```

Comment:
I am surprised that using the waku message timestamp here works. Cc @LNSD @danisharora099